app/policies/client_notification_policy.rb

```diff
@@ -0,0 +1,10 @@
+class ClientNotificationPolicy < ApplicationPolicy
+  # controller action policies
+  alias_method :index?,           :superadmin?
+  alias_method :clients_index?,   :superadmin?
```

Comment:
What's `clients_index` used for? The controller doesn't seem to have an action with that name.

Reply:
true! I was using it, that only a superadmin can see this button on the clients index, but not necessary indeed